cfme/tests/containers/test_logging.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+
+from cfme.containers.provider import ContainersProvider
+
+NUM_OF_DEFAULT_LOG_ROUTES = 2
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture(scope="function")
+def validate_logging_up_and_running(provider):
```

Comment:
Do we ensure that some logging is missing, and that some are off? I mean the on the provider itself?